apps/network-alerts/js/attention/attention.js

```diff
@@ -42,7 +42,10 @@ function sendNotification() {
         navigator.mozL10n.get(title), {
           body: body,
           tag: '' + Date.now(), // needs to be unique
-          icon: NotificationHelper.getIconURI(app) + '?titleID=' + title
+          icon: NotificationHelper.getIconURI(app),
+          data: {
```

Comment:
__issue:__ It's good start! So let's learn few more things :)

* (simplified) We have Treeherder that we can use to see results of unit/integration and all other types of tests that are run with your patch, you can find more info about it at [1]. So once you create your PR, Autolander [2] runs tests with your PR and pastes the link to the test results as a comment to your PR [3]. When we say ```Treeherder is green``` that means all tests are passing there. Take a look at [3]  we have two jobs that are failing because of your PR:
  * ```GU12``` - GU stands for "gaia unit tests" - you need to fix the unit test :) Use [4] as a guide and ask question if needed.
  * ```JSHINT``` - it's just jshint check that falied, it's not always easy to quickly understand if it's related to your patch, but you can click on the job, then click on "Open the raw log" button and figure out what exactly the reason.

* So you've changed the way we generate notification, good, but you should also change the way we use to parse this data back when user clicks on notification - I haven't tested, but the title of the message will be wrong when you tap on notification (via notification tray) to see network alert once again. Check this code [5] - when user taps on notification in notification tray, special "notification" system message is generated (argument is the notification you've created previously) that will wake up application and display alert message once again [6] - the change here will likely require related unit test change as well.

[1] https://developer.mozilla.org/en-US/docs/Mozilla/Firefox_OS/Automated_testing/Treeherder
[2] https://github.com/mozilla/autolander
[3] https://github.com/mozilla-b2g/gaia/pull/32418#issuecomment-147797476
[4] https://developer.mozilla.org/en-US/Firefox_OS/Automated_testing/Gaia_unit_tests
[5] https://github.com/mozilla-b2g/gaia/blob/master/apps/network-alerts/js/notification/notification.js#L15
[6] https://developer.mozilla.org/en-US/docs/Web/API/Navigator/mozSetMessageHandler